index.bs

```diff
@@ -17,12 +17,30 @@ Include MDN Panels: if possible
 Include Can I Use Panels: yes
 </pre>
 
-Introduction {#intro}
-=====================
+# Introduction # {#introduction}
 
-For now, see the [explainer]([REPOSITORYURL]).
+*This section is non-normative.*
 
-See [https://garykac.github.io/procspec/](https://garykac.github.io/procspec/),
-[https://dlaliberte.github.io/bikeshed-intro/index.html](https://dlaliberte.github.io/bikeshed-intro/index.html),
-and [https://speced.github.io/bikeshed/](https://speced.github.io/bikeshed/) to get started on your
-specificaton.
+A document can request the user agent that the default speculative parsing behavior is not used, by using the `Prefer-No-Speculative-Parsing` HTTP header. This header is a HTTP structured field whose value must be Boolean.
```

Comment:
This part is normative, especially the "must" sentence.

I think it's OK to remove "This section is non-normative" since it's implicit in how the paragraphs are written. It has one normative part and then a non-normative paragraph full of [statements of fact](https://ln.hixie.ch/?start=1140242962&count=1).